test/test_add_products.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+
+def test_add_product_to_db(client):
+    product_data = {
+        'name': 'Product Name',
+        'price': 9.99,
+        'type': 'food',
+        'region': 'Europe'
+    }
+
+    response = client.post('/products', json=product_data)
+    assert response.status_code == 201
+    assert response.json() == product_data
```

Comment:
You should also check that the product was really added to the database

Reply:
This comment remains. You have to retrieve the product from the DB and ensure that this project exists there!

Reply:
This test is a duplicate of the below's one